launch_frontend/launch_frontend/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Main entry point for the `launch_frontend` package."""
+
+from .entity import Entity
+from .parser import parse_executable, parse_node
+
+__all__ = [
+    'Entity',
+    'parse_executable',
+    'parse_node'
```

Comment:
@ivanpauno just for the record, it's ok to have these `parse_*()` functions around for testing, but they are too specific to be part of the `launch_frontend` package.